packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -230,7 +242,30 @@ export const reducer = handleActions({
     [CUT]: contextPath => $set('cr.nodes.clipboard', contextPath),
     [PASTE]: () => $set('cr.nodes.clipboard', ''),
     [HIDE]: contextPath => $set(['cr', 'nodes', 'byContextPath', contextPath, 'properties', '_hidden'], true),
-    [SHOW]: contextPath => $set(['cr', 'nodes', 'byContextPath', contextPath, 'properties', '_hidden'], false)
+    [SHOW]: contextPath => $set(['cr', 'nodes', 'byContextPath', contextPath, 'properties', '_hidden'], false),
+    [UPDATE_URI]: ({oldUri, newUri}) => state => {
+        const allNodes = $get('cr.nodes.byContextPath', state);
+        // Make sure to not include false positives by checking that the given segment ends either with "/" or "@"
+        const containsOldUriSegmentRegex = new RegExp(oldUri + '(/|@)');
```

Comment:
here, we should escape "oldUri" e.g. using https://lodash.com/docs#escapeRegExp, no?

Reply:
actually I would not use regexes here, as we only check for substring matching of two strings, right?

Reply:
yep, will do so